eamena/eamena/templates/views/forms/modification.htm

```diff
@@ -0,0 +1,92 @@
+{% extends "views/forms/base.htm" %}
+{% load i18n %}
+
+
+
+{% block form_content %}
+<div class="form-full-content">
+    <div class="row">
+        <div class="col-xs-12 margin-bottom-10">
+            <div class="form-section-header"><span>{% trans "Modifications" %}</span></div>
+        </div>
+    </div>
+    <div id="modification-section">
```

Comment:
@mradamcox This is not one section, but rather should be separated out into three, as the E24 Interface doc explains:

> 1. Type of Modification (Modification_Type.E55) & Date of Modification (Modification_Date.E49)
> 2. Modification Remarks (Modification_Description.E62) 
> 3. Construction Material (Construction_Material.E57) & Construction Technique (Construction_Technique_Type.E55) & Note (Construction_Note.E62) 


Each of the three above should be given a separate section